docs/csharp/quick-starts/numbers-in-csharp-local.md

```diff
@@ -351,6 +351,7 @@ the area of a circle whose radius is 2.50 inches. Remember that the area of a ci
 is the radius squared multiplied by PI. One hint: C# contains a constant
 for PI, <xref:System.Math.PI?displayProperty=nameWithType> that you can use for that value. 
 
+You should get an answer between 19 and 20 (remember that doouble arithmetic rounds.)
```

Comment:
One nit on line 351: C# contains a constant... But it's actually not a language feature. Perhaps ".NET includes a constant..."?
double is misspelled. But I'm not sure that the meaning of "remember that double arithmetic rounds" is clear in the context.

Reply:
I removed that sentence. I don't think it adds value.